test/Image3.rb

```diff
@@ -298,7 +298,7 @@ def test_resize_to_fit3
     keep = img
     img.resize_to_fit!(100)
     expect(img).to be_instance_of(Magick::Image)
-    assert_same(img, keep)
+    expect(keep).to be(img)
```

Comment:
Why is this being tested?

Reply:
Yeah, this doesn't make any sense. They're the same reference so there's no reason they would be different.